Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -1639,6 +1639,10 @@ void WebPage::bindRemoteAccessibilityFrames(int, WebCore::FrameIdentifier, Vecto
 {
 }
 
+void WebPage::resolveAccessibilityHitTestForTesting(const WebCore::IntPoint& point, CompletionHandler<void(String)>&& completionHandler)
```

Comment:
I think when we don't use function arguments at all, we don't give them a name. So:

```cpp
void WebPage::resolveAccessibilityHitTestForTesting(const WebCore::IntPoint&, CompletionHandler<void(String)>&&)
{
}
```